pkg/resourcetracker/app.go

```diff
@@ -20,22 +20,35 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/oam-dev/kubevela/pkg/monitor/metrics"
-
 	"github.com/crossplane/crossplane-runtime/pkg/meta"
+	"github.com/pkg/errors"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/oam-dev/kubevela/apis/core.oam.dev/common"
 	"github.com/oam-dev/kubevela/apis/core.oam.dev/v1beta1"
+	"github.com/oam-dev/kubevela/pkg/monitor/metrics"
 	"github.com/oam-dev/kubevela/pkg/oam"
+	velaerrors "github.com/oam-dev/kubevela/pkg/utils/errors"
 )
 
 const (
 	// Finalizer for resourcetracker to clean up recorded resources
 	Finalizer = "resourcetracker.core.oam.dev/finalizer"
 )
 
+var (
+	applicationResourceTrackerGroupVersionKind = schema.GroupVersionKind{
+		Group:   "apiextensions.core.oam.dev",
```

Comment:
```suggestion
		Group:   "prism.core.oam.dev",
```

Reply:
@wonderflow How about "prism.oam.dev"? Will it be confusing?

Reply:
prism.oam.dev is fine

Reply:
OK.